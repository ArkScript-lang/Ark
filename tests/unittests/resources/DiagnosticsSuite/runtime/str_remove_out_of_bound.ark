(str:removeAt "abc" 5)
