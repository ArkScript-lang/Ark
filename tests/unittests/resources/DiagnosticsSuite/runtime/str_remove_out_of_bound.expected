str:removeAt: index 5 out of range (length: 3)
